src/libutil/current-process.hh

```diff
@@ -14,16 +17,18 @@ namespace nix {
  */
 unsigned int getMaxCPU();
 
+#ifndef _WIN32 // TODO implement on Windows, if needed.
 /**
  * Change the stack size.
  */
 void setStackSize(rlim_t stackSize);
+#endif
```

Comment:
https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-setthreadstackguarantee to make unconditional

Reply:
#10540